neural_mfg.py

```diff
@@ -46,7 +46,7 @@ def __init__(self):
         
         self.gamma = var['mfg_params']['gamma']; # discount factor
         self.alpha = var['mfg_params']['alpha']; # conjestion operator
-        if gamma == 0:
+        if self.gamma > 0:
```

Comment:
Why did you change this?

Reply:
I was working on the branch before you updated the code with the fixed bug, the result should be the same, no?